src/components/ToSTIX/Mapping.jsx

```diff
@@ -0,0 +1,356 @@
+                                      <div key={fieldId + stixField.id}>
+                                        <div
+                                          className={`bx--row ${styles.object_item__field}`}
+                                        >
+                                          <div className={"bx--col-sm-1"}>
+                                            <TextInput
+                                              id={stixField.id + stixField.key}
+                                              labelText={""}
+                                              onChange={(e) => {
+                                                dispatch(
+                                                  updateMappingField(
+                                                    objectName,
+                                                    fieldId,
+                                                    stixField.id,
+                                                    e.target.value,
+                                                    "key"
+                                                  )
+                                                );
+                                              }}
+                                              size={"sm"}
+                                              value={stixField.key}
+                                            />
+                                          </div>
+                                          <div className={"bx--col-sm-1"}>
+                                            <Dropdown
+                                              size={"sm"}
+                                              ariaLabel="Dropdown"
+                                              id="transformer"
+                                              items={getTransformers()}
+                                              label="None"
+                                              selectedItem={
+                                                stixFieldTransformer
+                                              }
+                                              onChange={(e) => {
+                                                dispatch(
+                                                  updateMappingField(
+                                                    objectName,
+                                                    fieldId,
+                                                    stixField.id,
+                                                    e.selectedItem,
+                                                    "transformer"
+                                                  )
+                                                );
+                                              }}
+                                            />
+                                          </div>
+                                          <div className={"bx--col-sm-1"}>
+                                            <MultiSelect
```

Comment:
can we make the `none` appear only if nothing selected? I don't want the `none` to be selectable as in the following image
<img width="227" alt="image" src="https://user-images.githubusercontent.com/16198896/128630523-e9873b6b-a431-4e44-902e-7eee4805a64c.png">